frontend/app/components/comment/comment.tsx

```diff
@@ -84,12 +85,11 @@ export class Comment extends Component<Props, State> {
       scoreDelta: 0,
       cachedScore: props.data.score,
       initial: true,
+      ...this.updateState(),
```

Comment:
I accidentally committed a fix for warning about `this.setState` at `constructor`.